doc/source/train/user-guides/experiment-tracking.rst

```diff
@@ -1,195 +1,292 @@
-.. _train-experiment-tracking:
+.. _train-experiment-tracking-native:
 
-Experiment Tracking
-===================
+==================================
+Experiment Tracking with Ray Train
```

Comment:
```suggestion
Experiment Tracking
```
<img width="292" alt="Screenshot 2023-08-23 at 3 28 55 PM" src="https://github.com/ray-project/ray/assets/26745457/23e5a932-8551-426a-ab2a-563f51cf1e3d">

Is this more consistent with other titles?

Reply:
good point. updated!